packages/synthetic-chain/src/lib/constants.js

```diff
@@ -13,3 +13,5 @@ export const CHAINID = process.env.CHAINID;
 export const HOME = process.env.HOME;
 
 export const SDK_ROOT = '/usr/src/agoric-sdk';
+
+export const GAS_FLAGS = '--gas=auto --gas-adjustment=1.2'
```

Comment:
why isn't `gas=auto` sufficient? 

Are these the only GAS_FLAGS we'd ever use?

If so, should this just be part of the `agd.tx` command?

Reply:
> why isn't `gas=auto` sufficient?

In practice I've observed it necessary, `.simulate()` seems to often undershoot. [cosmos-sdk docs](https://hub.cosmos.network/main/hub-tutorials/gaiad.html#:~:text=You%20may%20want,value%20is%201.0) give an explanation, as does [this issue](https://github.com/cosmos/cosmos-sdk/issues/4938)

> Are these the only GAS_FLAGS we'd ever use?

There's also `--gas-prices=[0.0001ubld]`, but it doesn't seem like this is currently needed (maybe it will be needed when nodes start setting minimum gas prices?)

> should this just be part of the agd.tx command

I'll defer to your guidance here, happy to move this to `agd.tx` in `agd-lib` and `cliHelper`.

Reply:
If it's a good default, then I think it better to be in `agd.tx`. There is some precedent with`-bblock`. 
https://github.com/Agoric/agoric-3-proposals/blob/40c0cf674fd2d72593d91ea9db87f06b264f6265/packages/synthetic-chain/src/lib/cliHelper.js#L19-L22

Aside, I just noticed that that one's being passed in redundantly.

If there are cases where we _don't_ want these gas settings then I agree it should be a constant but I think the name should describe the use case so the callsite documents why it's being included.  E.g. `AUTO_GAS`

Reply:
Updated to include `--gas auto --gas-adjustment 1.3` in [7d6a5e2](https://github.com/Agoric/agoric-3-proposals/pull/73/commits/7d6a5e212f77db5a45e4765e3aa17a98e51b15de).